### ISSUE TITLE
dotnet-bot image doesn't load on Windows unpackaged app

### ISSUE THREAD

**Issue by eerhardt** [dotnet/maui#6202]
### Description

When I run a `dotnet new maui` app on Windows in an unpackaged app, I don't see the dotnet-bot image.

### Steps to Reproduce

1. `dotnet new maui`
2. `dotnet publish -f net6.0-windows10.0.19041.0 -c Release -p:WindowsPackageType=None`
3. `bin\Release\net6.0-windows10.0.19041.0\win10-x64\publish\MauiTest.exe`

I don't see the dotnet-bot image. Instead the app looks like:

![image](https://user-images.githubusercontent.com/8291187/163875843-6d6f3edf-2bb3-4827-9382-13b7a4cabcb5.png)

### Version with bug

Unknown/Other (please specify)

### Last version that worked well

Unknown/Other

### Affected platforms

Windows

### Affected platform versions

Windows 10

**Comment by kristinx0211** [comment]
Duplicate of #5927.

**Comment by XamlTest** [comment]
Verified this issue with Visual Studio Enterprise 17.3.0 Preview 1.0 [32427.455.main]. Repro on Windows.

**Comment by jfversluis** [comment]
Did you also run into the fact that you can't close the app with the X in the top-right?

**Comment by MaxWang-MS** [comment]
Hi @jfversluis, I can confirm I ran into the same problem as above. I installed the GA version of Visual Studio 17.3 along with Maui that comes with it. I did not install Maui preview on the machine before. After following the repro steps in the original post above, the bot image also did not come up for me when running the exe in the `publish` folder. The title bar was working for me though - I was able to minimize or close the app.

**Comment by mobycorp** [comment]
I completely disagree with the decision to move this to the graveyard, otherwise known as the backlog! This is a CRUSHING error!!! I followed @jfversluis's YouTube video on how to publish an unpackaged deployment and NONE of the images show! None of the FontImageSource are rendered either. I am hoping I can get a response in the next day or so telling me the solution to this grave problem!! This is a complete deal breaker!

**Comment by jfversluis** [comment]
There has been a lot of work going on to support unpackaged apps. Not sure into what versions this all went, but I just tried it on .NET 7 and it works fine for me there. If this is important to you, please update to all the latest bits and use that.

If you still think that doesn't work feel free to open a new issue with all the details of your specific scenario. 

The one described here, assuming you're using .NET 7, works as intended now from what I can see.

**Comment by mobycorp** [comment]
Hi Gerald,

I am using .NET 7 with Visual Studio Version 17.4.2. Obviously, it is VERY
important that my icons and font icon images display! I followed your
examples and, with the exception of the overlooked issue I posted a comment
on your YouTube video for this topic indicating that you also have to
set PublishAppxPackage to false in the publish profile. Here is what it
looks like packaged:

[image: image.png]
And, this is what it looks like unpackaged with nothing changed except the
three lines of code you have published:

[image: image.png]

Not only this, but I have reported at least 10 issues and they all still
exist. I have been working on the backend of the app implementing a gRPC
P2P app and have not had my fingers on the UI code until now. Look at the
title bar... I have reported this, you guys fixed it, then when a new
version of VS rolled out, it was broken again. Very frustrating in that I
am trying to get a demo for an investor who is potentially going to invest
a huge sum of money and I can't show the app like this!

So, no, it does not work as designed.

I will open a new issue.

Thanks!
Steve
